constants.go

```diff
@@ -50,9 +50,9 @@ const (
 	// SamplerParamTagKey reports the parameter of the sampler, like sampling probability.
 	SamplerParamTagKey = "sampler.param"
 
-	// TracerStateHeaderName is the http header name used to propagate tracing context.
+	// TraceContextHeaderName is the http header name used to propagate tracing context.
 	// This must be in lower-case to avoid mismatches when decoding incoming headers.
-	TracerStateHeaderName = "uber-trace-id"
+	TraceContextHeaderName = "uber-trace-id"
```

Comment:
Please also keep the old constant with a comment `// Deprecated: use TraceContextHeaderName`